src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -133,6 +133,47 @@ private static ClientInfo CreateClientInfo(InputClient ns, SourceInputModel? sou
             return new ClientInfo(ns.Name, clientName, clientNamespace, clientDescription, existingType, operations, clientParameters, resourceParameters);
         }
 
+        private static IReadOnlyList<InputOperation> UpdateOperations(IReadOnlyList<InputOperation> originalOperations)
```

Comment:
If this change is applied to `InputOperation`, we should be able to call this at the beginning of `Build` method, after or before `UpdateListMethodNames`.